docs/current_docs/quickstart/183917-daggerverse.mdx

```diff
@@ -0,0 +1,18 @@
+---
+slug: /quickstart/183917/daggerverse
+hide_table_of_contents: true
+title: "Use modules from the Daggerverse"
+---
+
+# Quickstart
+
+## Use modules from the Daggerverse
+
+As you've seen in previous sections, modules can be consumed from remote sources like GitHub, or installed and used locally. The [Daggerverse](https://daggerverse.dev) is an online catalog of Dagger modules that enables users to easily discover and use modules created by the Dagger community.
+
+All Dagger CLI commands work as expected with Daggerverse URLs. Here are a few more examples for you to try:
```

Comment:
This is confusing, it makes it seem like there are "Daggerverse URLs" as opposed to "non-Daggerverse URLs".

In fact Daggerverse does not affect URLs at all: it just indexes all modules so that you can find them.

Reply:
I saw you updated this, marking as resolved